src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -373,7 +374,7 @@ public static RavenConfiguration CreateForDatabase(RavenConfiguration parent, st
         /// </summary>
         internal static RavenConfiguration CreateForTesting(string name, ResourceType resourceType, string customConfigPath = null)
         {
-            return new RavenConfiguration(name, resourceType, customConfigPath);
+            return new RavenConfiguration(name, resourceType, customConfigPath, skipEnvironmentVariables: true);
```

Comment:
It fails the test if you are using 

![image](https://user-images.githubusercontent.com/116915/119640159-ff83f100-be20-11eb-803a-79f2b29cae4e.png)

Reply:
FYI, a lot of tests will fail if you will set RAVEN.Indexing.Disabled to true. IMO we should revert this. This is local problem (of your machine) and now we made it a potential global problem.

Reply:
The problem is that this is something that we _need_ if we want to debug secured tests.
Otherwise, this creates a lot of hassle.
This function is used in 3 locations, and this is modified only for this test.